src/main/java/repository/OwnerLoader.java

```diff
@@ -0,0 +1,48 @@
+package repository;
+
+import domain.*;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by roberj78 on 06/10/2016.
+ */
+public class OwnerLoader {
```

Comment:
Loader could be ambiguous - maybe call it Extractor as more consistent with common naming practices